sycl/source/detail/scheduler/commands.cpp

```diff
@@ -81,14 +81,26 @@ static std::string accessModeToString(access::mode Mode) {
   }
 }
 
+static std::vector<RT::PiEvent>
+getPiEvents(const std::vector<EventImplPtr> &EventImpls) {
+  std::vector<RT::PiEvent> RetPiEvents;
+  for (auto &EventImpl : EventImpls)
+    RetPiEvents.push_back(EventImpl->getHandleRef());
+  return RetPiEvents;
+}
```

Comment:
You can replace this with `std::transform`

Reply:
@AlexeySachkov

Reply:
> In this file, the functionality is called at multiple locations..

I will leave it up to you, then. Not sure that inserting `std::transform` into this helper will help a lot, so, you can leave it as is